src/parser/jobs/ast/modules/Lightspeed.tsx

```diff
@@ -0,0 +1,44 @@
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import CastTime from 'parser/core/modules/CastTime'
+import {Data} from 'parser/core/modules/Data'
+
+const LIGHTSPEED_REDUCTION = -2500
+
+// in this module we only want to track Lightspeed windows for castTime/weaving purposes
+export class Lightspeed extends Analyser {
+	static override handle = 'Lightspeed'
+
+	@dependency private data!: Data
+	@dependency private castTime!: CastTime
+
+	private castedTime: number | undefined = undefined //when lightspeed was casted
```

Comment:
@otocephaly 
Might be worth cleaning this up in a follow-up, since this is really going unused.  I realize it was probably lifted from the previous incarnation of the code and so I'm not going to block on it, but might be worth removing the vestigal code so future maintainers don't have to wonder what's going on here.

Reply:
oop I thought I cleaned it up
I'll keep in mind for a clean up in the near future then
thanks for the advice